src/scenes/endscreen.ts

```diff
@@ -30,17 +32,40 @@ export class EndScreen extends Phaser.Scene {
         this.scene.start("TitleScreen");
     }
 
-    private displayMessage(): void {
-        let text: string = "You loose";
+    private displayByRole(): void {
+        const { showerOre, text } = this.displayData();
+        this.createShower(showerOre);
+        this.displayText(text);
+    }
+
+    private displayData() {
+        let text: string = "You loose!";
+        let showerOre: string = "coalNugget";
         const me: Nullable<Player> = PlayerManager.getMe();
         assert(me);
         if (
             (me.role === Role.Miner && SessionManager.getReachedGold()) ||
             (me.role === Role.Saboteur && !SessionManager.getReachedGold())
         ) {
-            text = "You win baby";
+            showerOre = "goldNugget";
+            text = "You win!";
         }
+        return { showerOre, text };
+    }
+
+    private createShower(showerOre: string): void {
+        this.add.particles(0, 100, showerOre, {
+            x: { min: 0, max: ScreenWidth },
+            y: -250,
+            accelerationY: 300,
+            maxVelocityY: 300,
+            quantity: 1,
+            lifespan: 5000,
+            gravityY: 200,
+        });
```

Comment:
Visual input: Rotating falling down & maybe different scale to simulate depth?

Reply:
Is this something we should focus our time on?

Reply:
Its one line each, exactly like the other properties:
```
scale: { min: 0.2, max: 0.8 },
rotate: { start: 0, end: 360 },
```

Reply:
Uhhh wow, looks amazing! 🔥